src/test/resources/shopping-list-no-key.json

```diff
@@ -0,0 +1,92 @@
+{
+  "slug": {
+    "en": "shoppingList-slug-1"
+  },
+  "name": {
+    "en": "shoppingList-name-1"
+  },
+  "description": {
+    "en": "description-1"
+  },
+  "customer": {
+    "typeId": "customer",
+    "id": "368ddf13-cd6c-4ff4-ad38-73038aeed5ba"
+  },
+  "lineItems": [
```

Comment:
Just leave 1 lineItem.

Reply:
created a new PR and please check this commit for the fix. https://github.com/commercetools/commercetools-project-sync/pull/213/commits/8030e3bdb0a9252deeb2e8a2b519333cc58d0861